applications/tools/p2p-test/app-simulator/charts/app-simulator/values.yaml

```
@@ -30,7 +30,11 @@ resources:
   requests: {}
 
 # -- node labels for pod assignment, see https://kubernetes.io/docs/user-guide/node-selection/
-nodeSelector: {}
+nodeSelector: { node: "prereqs" }
```

nit: although allowable, generally it's better not to mix YAML and JSON (the exception is the use of `{}` to indicate that it's expecting an object here which you can't convey in YAML)
```suggestion
nodeSelector:
  node: "prereqs"
```